tests/test.ts

```diff
@@ -1,24 +1,24 @@
 import { Project } from 'fixturify-project';
```

Comment:
is this a CJS TS? (I noticed there was an mjs -- porque no mts?)

Reply:
the mjs is just for a test to make sure that it can explicitly be imported correctly from a mjs file. I don't have any opinion on ts or mts really 🤷 other than to say that this PR adds `type: module` to the package.json so it's not required (in the same way that you only need mjs when you're not using `type: module` in package.json)